mmhuman3d/models/backbones/hrnet.py

```diff
@@ -766,3 +771,256 @@ def forward(self, x):
         xf = xf.mean(dim=(2, 3))
         xf = xf.view(xf.size(0), -1)
         return xf
+
+
+class PoseHighResolutionNetPyMAFX(BaseModule):
+    """HRNet backbone for pymaf-x."""
+    blocks_dict = {'BASIC': BasicBlock, 'BOTTLENECK': Bottleneck}
+
+    def __init__(self,
+                 extra,
+                 pretrained=True,
+                 global_mode=False,
+                 init_cfg=None):
+        self.inplanes = 64
+        super(PoseHighResolutionNetPyMAFX, self).__init__(init_cfg)
+
+        # stem net
+        self.conv1 = nn.Conv2d(
```

Comment:
Shall we try to use [build_conv_layer](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/models/backbones/hrnet.py#L78) and [build_norm_layer](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/models/backbones/hrnet.py#L85)?